src/utils/FormInputHandling.ts

```diff
@@ -17,14 +17,65 @@ async function getData(currentPage: number, limit: number) {
 
 // post data to the server
 async function postFormData(formData: RowType) {
-  const currentDate = new Date().toLocaleDateString();
+  const { student, test, timeline, dateCreated } = formData;
+  let start_time = timeline.startDate + "T" + timeline.startTime + ":00";
+  let end_time = timeline.endDate + "T" + timeline.endTime + ":00";
+  const requestBody = {
+    name: test.name,
+    platform: test.platform,
+    platform_link: test.platformLink,
+    portal_link: test.portalLink,
+    start_time,
+    end_time,
+    is_active: "",
+    purpose: {
+      type: "attendance",
+      params: "quiz",
+    },
+    repeat_schedule: { type: "weekly", params: [1, 2, 3, 4, 5, 6, 7] }, // this is hardcoded needs to be changed
+    session_id: "",
+    platform_id: "",
+    form_schema_id: "1",
```

Comment:
comment where this would go eventually. its a session parameter. probably  `// session.form_schema_id` 

i feel like we'll have another formtype corresponding to session.. that will have things like .. - form_schema_id, type, auth type, activate signup, id generation, redireciton, pop up form, number of fields in pop up form, etc

Reply:
added` `SessionTypes``